.github/workflows/build-deb.yaml

```diff
@@ -69,6 +69,10 @@ jobs:
                 cargo-vendor-filterer@${{ env.CARGO_VENDOR_FILTERER_VERSION }}
               command -v cargo-vendor-filterer
             fi
+            # Install Go 1.25 from our Go backports PPA
+            sudo add-apt-repository ppa:ubuntu-enterprise-desktop/golang
```

Comment:
FYI I've added this to https://launchpad.net/~ubuntu-enterprise-desktop/+archive/ubuntu/authd-edge/+edit-dependencies and friends, otherwise we'd fail on build there